ci/build_setup.ps1

```diff
@@ -18,4 +18,4 @@ echo "ENVOY_BAZEL_ROOT: $env:ENVOY_BAZEL_ROOT"
 echo "ENVOY_SRCDIR: $env:ENVOY_SRCDIR"
```

Comment:
For the maintainers to consider: this file is not actually used in CI and only is referenced in `do_ci.ps1`, should we change over our CI pipelines to use the `do_ci` pattern like on Linux? (or should we just simply keep it around until we can do the full set of builds/tests?)

Reply:
we're deferring changing around how CI is done on Windows for now